cli/pkg/printer/resources.go

```diff
@@ -231,9 +253,10 @@ type memberUser struct {
 }
 
 type memberUserWithRole struct {
-	Email    string `header:"email" json:"email"`
-	Name     string `header:"name" json:"display_name"`
-	RoleName string `header:"role" json:"role_name"`
+	Email      string `header:"email" json:"email"`
+	Name       string `header:"name" json:"display_name"`
+	RoleName   string `header:"role" json:"role_name"`
+	Attributes string `header:"attributes" json:"attributes"`
```

Comment:
Can you make it hide this column for project listings (where there are no attributes)?

Reply:
Absolutely